packages/phishing-controller/src/PhishingDetector.test.ts

```diff
@@ -75,87 +75,169 @@ describe('PhishingDetector', () => {
         );
       });
 
-      [
-        undefined,
-        null,
-        true,
-        false,
-        0,
-        1,
-        1.1,
-        '',
-        () => {
-          return { name: 'test', version: 1 };
-        },
-        {},
-      ].forEach((mockInvalidName) => {
-        it('throws an error when config name is invalid', async () => {
-          await expect(
-            withPhishingDetector(
-              [
-                {
-                  allowlist: [],
-                  blocklist: ['blocked-by-first.com'],
-                  fuzzylist: [],
-                  // @ts-expect-error testing invalid input
-                  name: mockInvalidName,
-                  tolerance: 2,
-                  version: 1,
-                },
-              ],
-              async () => mockInvalidName,
-            ),
-          ).rejects.toThrow("Invalid config parameter: 'name'");
-        });
-      });
+      it('logs an error when config name is invalid for various invalid inputs', async () => {
```

Comment:
This test would be much better if we leverage the [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout)